packages/core/src/SlideBox/Slide.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef, useEffect } from 'react'
+import PropTypes, { InferProps } from 'prop-types'
+import { Box } from '..'
+import { RenderInView } from './RenderInView'
+import { SlideWrapper } from './styles'
+
+const propTypes = {
+  onSlideChange: PropTypes.func,
+  slideSpacing: PropTypes.number,
+  stretchHeight: PropTypes.bool,
+  index: PropTypes.number,
+  content: PropTypes.node,
+  isCurrentSlide: PropTypes.bool,
+  width: PropTypes.any,
+  numSlides: PropTypes.number,
+}
```

Comment:
Would you mind updating this to use both the propTypes object and an interface? Here's an example: https://github.com/priceline/design-system/blob/main/packages/core/src/Flex/Flex.tsx#L22

Reply:
No problem. just made the update